distribution/packages/src/rpm/wazuh-indexer.rpm.spec

```diff
@@ -204,6 +205,15 @@ if command -v systemd-tmpfiles > /dev/null; then
     systemd-tmpfiles --create %{name}.conf
 fi
 
+if [ -f %{tmp_dir}/wazuh-indexer.restart ]; then
+    rm -f %{tmp_dir}/wazuh-indexer.restart
```

Comment:
Consider using [systemd-tmpfiles](https://www.baeldung.com/linux/systemd-tmpfiles-configure-temporary-files)

Reply:
Analysis in the comment https://github.com/wazuh/wazuh-indexer/pull/492#issuecomment-2441861588